imctools/scripts/exportacquisitioncsv.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import os
+import pandas as pd
+import imctools.io.mcdxmlparser as mcdmeta
+
+SUFFIX_ACMETA = '_' + mcdmeta.ACQUISITION + mcdmeta.META_CSV
+COL_MCD_ID = mcdmeta.ID
+COL_ACID = mcdmeta.ACQUISITIONID
+COL_ACSESSION = 'AcSession'
+SUF_CSV = '.csv'
+AC_META = 'acquisition_metadata'
+
+VAL_TEMP = 'temp'
+"""
+Reads all the acquisition metadata from the ome folders and concatenates them
+to a csv that contains all the metadata.
+"""
+
+def _read_and_concat(fol_ome, suffix, idname):
+    ac_names = os.listdir(fol_ome)
+    dat = pd.concat([pd.read_csv(os.path.join(fol_ome, a, a+suffix)) for a in ac_names],
+                           keys=ac_names, names=[COL_ACSESSION, VAL_TEMP])
+    dat = dat.reset_index(VAL_TEMP, drop=True)
+    dat = dat.reset_index()
+    dat = dat.rename(columns={COL_MCD_ID: COL_ACID})
+    return dat
+
+def read_acmeta(fol_ome):
+    dat_acmeta = _read_and_concat(fol_ome, SUFFIX_ACMETA, COL_ACID)
+    return dat_acmeta
+
+def export_acquisition_csv(fol_ome, fol_out, outname=None):
```

Comment:
Thanks for pointing it out, you're surely right! 
Somehow I think when I wrote this I somehow mixed up and considered strings to be mutable in Python, in which case an initialization like this would be reasonable. As they are not, your way is clearer and the recommended one for sure!
I will change it accordingly.